index.html

```diff
@@ -15,8 +15,78 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
-  </header>
+
+  <header></header>
+
+
+  <div class="container">
+
+
+          <div class="row header-div">
+            <div class="column-half">
+             <h3 class="head-box-text">Code Journal</h3>
+            </div>
+          </div>
+
+
+      <div class="form-div" data-view="entry-form">
```

Comment:
You'll be putting this into another element with the `container` class once you make the above adjustment I mentioned.